src/preview.js

```diff
@@ -1,51 +1,70 @@
-import countries from '../assets/countries.json';
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+ import countries from '../assets/countries.json';
 import continentNames from '../assets/continent-names.json';
 import continents from '../assets/continents.json';
 import { getEmojiFlag } from './utils';
-import { __, sprintf } from '@wordpress/i18n';
 
 export default function Preview( { countryCode, relatedPosts } ) {
 	if ( ! countryCode ) return null;
 
 	const emojiFlag = getEmojiFlag( countryCode ),
-	      hasRelatedPosts = relatedPosts?.length > 0;
+		hasRelatedPosts = relatedPosts?.length > 0;
 
 	return (
 		<div className="xwp-country-card">
-			<div className="xwp-country-card__media" data-emoji-flag={ emojiFlag }>
-				<div className="xwp-country-card-flag">
-					{ emojiFlag }
-				</div>
+			<div
+				className="xwp-country-card__media"
+				data-emoji-flag={ emojiFlag }
+			>
+				<div className="xwp-country-card-flag">{ emojiFlag }</div>
 			</div>
 			<h3 className="xwp-country-card__heading">
-				{ __( 'Hello from' ) }
-				{ ' ' }
-				<strong>{ countries[countryCode] }</strong>
-				{ ' ' }
-				(<span className="xwp-country-card__country-code">{ countryCode }</span>),
-				{ ' ' }
-				{ continentNames[continents[countryCode]] }!
+				{ __( 'Hello from', 'xwp-country-card' ) }{ ' ' }
+				<strong>{ countries[ countryCode ] }</strong> (
+				<span className="xwp-country-card__country-code">
+					{ countryCode }
+				</span>
+				), { continentNames[ continents[ countryCode ] ] }!
```

Comment:
This is very problematic. It wouldn't be easy to translate since the actual heading is very fragmented and may not make sense in such a form in other languages.

Gutenberg has a function that can be used to handle such cases better:

 ```js
{ createInterpolateElement(
	sprintf(
		// translators: %1$s: is a country name. %2$s: is a country code (e.g. US). %3$s: is a continent name.
		__(
			'Hello from <CountryName>%1$s</CountryName> (<CountryCode>%2$s</CountryCode>), %3$s!',
			'xwp-country-card'
		),
		countries[ countryCode ],
		countryCode,
		continentNames[ continents[ countryCode ] ]
	),
	{
		CountryName: <strong />,
		CountryCode: (
			<abbr
				className="xwp-country-card__country-code"
				title={ countries[ countryCode ] }
			/>
		),
	}
) }
```

Also note that the plain `span` has been replaced with an `abbr` so that the country code is better handled on the frontend.